.travis.yml

```diff
@@ -95,7 +90,7 @@ script:
    travis_wait 30 valgrind --error-exitcode=42 ./exhaustive_tests;
    fi
  - if [ -n "$CTIMETEST" ]; then
-   libtool --mode=execute valgrind  ./valgrind_ctime_test &> valgrind_ctime_test.log;
+   ./libtool --mode=execute valgrind  ./valgrind_ctime_test &> valgrind_ctime_test.log;
```

Comment:
Maybe just invoke valgrind_ctime_test.sh here?

Reply:
Oh, right and that's actually the correct thing to do because it adds the `--error-exitcode=1` parameter. 

Without that, valgrind simply passes the exit code through, even in the case of valgrind errors. That is, travis would succeed even if valgrind found problems. I fixed this and added a note to the commit message.

Reply:
Have you actually tested this? e.g. add a constant time error (perhaps by reverting a recent change) and seen if travis will fail?

Reply:
Good call. I tested locally that it outputs the right exit code but apparently it does not fail on travis for reasons I don't understand yet: -.-
https://travis-ci.org/real-or-random/secp256k1/jobs/659057756#L434

On top of that, we don't need to call the test in travis because it's included in make check now anyway.

I'll fix that stuff.

Reply:
That also may mean that none of the other valgrind tests in travis are doing anything.

Reply:
> That also may mean that none of the other valgrind tests in travis are doing anything.

I don't think so because the other valgrind tests have `--error-exitcode=42`

Reply:
> Good call. I tested locally that it outputs the right exit code but apparently it does not fail on travis for reasons I don't understand yet: -.-
> https://travis-ci.org/real-or-random/secp256k1/jobs/659057756#L434

For future reference: This was fixed by storing the exit code in an explicit variable instead of relying on `$?` twice. Apparently shells behave differently when we use `$?` in the case match and in the cases then.